Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h

```diff
@@ -614,6 +619,8 @@ class WebsiteDataStore : public API::ObjectImpl<API::Object::Type::WebsiteDataSt
     FileSystem::Salt m_mediaKeysStorageSalt;
 
     bool m_isBlobRegistryPartitioningEnabled { false };
+
+    std::optional<HashMap<WebCore::RegistrableDomain, RestrictedOpenerType>> m_restrictedOpenerTypesForTesting;
```

Comment:
Not convinced it is worth using std::optional here. an empty HashMap is basically a null pointer. Can't you just check if the HashMap is empty instead of checking for nullopt?